documentation/asciidoc/computers/configuration/headless.adoc

```diff
@@ -8,58 +8,20 @@ If you already have an image on the micro SD card, you can access the xref:confi
 
 If you plan on using VNC to access the desktop environment of your headless Raspberry Pi, use the `raspi-config` tool to set a xref:configuration.adoc#resolution[default display resolution].
 
-IMPORTANT: If you are installing Raspberry Pi OS, and intend to run it headless, you will need to create a new user account. Since you will not be able to create the user account xref:getting-started.adoc#configuration-on-first-boot[using the first-boot wizard] as it requires both a monitor and a keyboard, you *MUST* add a `userconf.txt` file to the boot folder to create a user on first boot or configure the OS with a user account using the xref:getting-started.adoc#advanced-options[Advanced Menu] in the Raspberry Pi Imager.
+IMPORTANT: If you are installing Raspberry Pi OS, and intend to run it headless, you will need to create a new user account. Since you will not be able to create the user account xref:getting-started.adoc#configuration-on-first-boot[using the first-boot wizard] as it requires both a monitor and a keyboard, you *MUST* add a `userconf.txt` file to the boot folder to create a user on first boot or configure the OS with a user account using the xref:getting-started.adoc#advanced-options[Advanced Menu] in Raspberry Pi Imager.
```

Comment:
Along the lines of my other comment, I think we should mention RPi Imager more prominently (as the recommeded approach), rather than only mentioning it at the end.

Reply:
OK! I just made a minor wording tweak here, but I can update that logic in this section as well.

Reply:
The reason it's written the way it is, is that when this feature was first added, RPi Imager initially didn't have the option for configuring the username - that only came later.